docker/nginx/server.conf

```diff
@@ -0,0 +1,37 @@
+server {
+    listen 80;
+
+    server_name localhost;
+    root /var/www/web;
+
+    location / {
+        try_files $uri /app.php$is_args$args;
+    }
+
+    # DEV
+    location ~ ^/(app_dev|config)\.php(/|$) {
+        fastcgi_pass php:9000;
+        fastcgi_split_path_info ^(.+\.php)(/.*)$;
+        include fastcgi_params;
+        fastcgi_param SCRIPT_FILENAME $realpath_root$fastcgi_script_name;
+        fastcgi_param DOCUMENT_ROOT $realpath_root;
+    }
+
+    # PROD
+    location ~ ^/app\.php(/|$) {
+        fastcgi_pass php:9000;
+        fastcgi_split_path_info ^(.+\.php)(/.*)$;
+        include fastcgi_params;
+        fastcgi_param SCRIPT_FILENAME $realpath_root$fastcgi_script_name;
+        fastcgi_param DOCUMENT_ROOT $realpath_root;
+        internal;
+    }
+
+    # Return 404 for other php files
+    location ~ \.php$ {
+        return 404;
+    }
+
+    error_log /var/log/nginx/neko_wiki.log;
+    access_log /var/log/nginx/neko_wiki.log;
```

Comment:
That's great. What is bad is that it's not reported outside of the docker. 2 solutions:
- configure (or check if it's not already the case?) to report on stderr (so it can be log by `docker-compose logs`)
- share the log directory with `var/logs` (and don't forget to add it in the `.gitignore`)

Reply:
I'll add it to `var/logs` folder